### DISCUSSION TITLE
How to change zookeeper node

### DISCUSSION THREAD

**Discussion by emahdij** [strimzi/strimzi-kafka-operator#6850] [Q&A] [answered]
Hi
I have a Kafka cluster that is managed with strimzi operator. I have 5 instances of Kafka and zookeeper. because of using topolvm, 
the zookeeper is limited to the node that PVC resides. I need to move one of the zookeeper instances.
I have a question: if I delete one of the PVC and zookeeper instances, will the new zookeeper be able to join the cluster and the cluster stays healthy?

**Comment by scholzj** [comment, marked as answer]
Yes, that should work. You need to make sure to not loose the quorum (majority of the ZooKeeper nodes needs to be online and synced). But as long as you don't do that, the new node should simply joind the cluster and recover the data from the remaining nodes.
